Week-3/exercise3.1/memoize.js

```diff
@@ -0,0 +1,27 @@
+function add(a = 0, b = 0) {
+    return a + b;
+}
+
+
+const memoize = (fn) => {
+    const cache = {};
+    return function (a, b) {
+        if (((a + '+' + b) in cache)) {
```

Comment:
this will work only for 2 arguments? refactor this code for n arguments

Reply:
function is updated or refactored to take n arguments.